### DISCUSSION TITLE
Make it possible to sort x-axis in Column chart, by sorting data before pass it into plot instance

### DISCUSSION THREAD

**Discussion by mhamrah** [antvis/G2Plot#2365] [Q&A] [answered]
* **G2Plot Version**: 2.3.7
<img width="1236" alt="image" src="https://user-images.githubusercontent.com/91162/104619439-dcb82380-565b-11eb-8ee9-26dd2e859c53.png">

I'd like to reverse the sort order of x-axis values in the column plot. Is this possible?

Thank you.

**Comment by hustcc** [comment, marked as answer]
This is a bug of scale. Now sort your data before pass it into plot instance is the best way.
